### ISSUE TITLE
References

### ISSUE THREAD

**Issue by italobusi** [haomianzheng/IETF-ACTN-YANG-Model#113]
Better to have a common style to reference ITU-T and other SDO standards

See: https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-flexigrid-yang/issues/41

OLD

```
   [ANSI]     American National Standards Institute, "Synchronous
              Optical Network (SONET) Basic Description including
              Multiplex Structure, Rates, and Formats", ANSI T1.105,
              January 1995.

   [IEEE]     Institute of Electrical and Electronics Engineers, "IEEE
              Standard for Ethernet", IEEE 802.3-2018, June 2018.

   [ITU-Tg7044]
              International Telecommunication Union, "Hitless adjustment
              of ODUflex(GFP)", ITU-T G.7044, October 2011.

   [ITU-Tg709]
              International Telecommunication Union, "Interfaces for the
              optical transport network", ITU-T G.709, June 2020.

   [ITU-Tgsup43]
              International Telecommunication Union, "Transport of IEEE
              10GBASE-R in optical transport networks (OTN)",
              ITU-T G.sup43, November 2011.

```
NEW

```
   [ANSI_T1.105]     American National Standards Institute, "Synchronous
              Optical Network (SONET) Basic Description including
              Multiplex Structure, Rates, and Formats", ANSI T1.105,
              January 1995.

   [IEEE_802.3]     Institute of Electrical and Electronics Engineers, "IEEE
              Standard for Ethernet", IEEE 802.3-2018, June 2018.

   [ITU-T_G.7044]
              International Telecommunication Union, "Hitless adjustment
              of ODUflex(GFP)", ITU-T G.7044, October 2011.

   [ITU-T_G.709]
              International Telecommunication Union, "Interfaces for the
              optical transport network", ITU-T G.709, June 2020.

   [ITU-T_G.Sup43]
              International Telecommunication Union, "Transport of IEEE
              10GBASE-R in optical transport networks (OTN)",
              ITU-T G.sup43, February 2011.

```

Note: the reference to G.Sup43 may be moved to the informative references depending on #110

**Comment by italobusi** [comment]
Already done in [draft-ietf-ccamp-layer1-types-15](https://datatracker.ietf.org/doc/html/draft-ietf-ccamp-layer1-types-15)
